worker/dbaccessor/manifold.go

```diff
@@ -106,8 +115,14 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				return nil, errors.Trace(err)
 			}
 
+			var slowQueryLogger coredatabase.SlowQueryLogger
+			if err := context.Get(config.QueryLoggerName, &slowQueryLogger); err != nil {
+				config.PrometheusRegisterer.Unregister(metricsCollector)
+				return nil, err
+			}
+
 			cfg := WorkerConfig{
-				NodeManager:      database.NewNodeManager(agentConfig, config.Logger),
+				NodeManager:      database.NewNodeManager(agentConfig, config.Logger, slowQueryLogger),
```

Comment:
I'm still thinking about how we can merge these two loggers into one. I'll sleep on it and then decide either way.

Reply:
Although they're both call loggers, they are in fact two different things. I don't think we should merge them just because they're named the same. We end up with cross-cutting concerns then.